### PR TITLE
Pmm 5434 Update QAN UI tests to check Information for Tables in PG

### PR DESCRIPTION
**Pull request by yurkovychv**
added tests for Postgres in QAN, tests for Updating/Clearing Alertmanager rules, updated change Metrics resolution test (removed Change Data Retention step)

### CODE REVIEW

File: tests/pages/pmmSettingsPage.js [hunk 1]
```diff
@@ -191,7 +203,7 @@ module.exports = {
 
     customClearField(field) {
         I.appendField(field, '');
-        I.pressKey(['Shift', 'Home']);
+        I.pressKey(["Control", "a"]);
```
Comment (puneet0191) [inline tests/pages/pmmSettingsPage.js]: why

File: tests/verifyQAN_test.js [hunk 2]
```diff
@@ -46,4 +46,52 @@ Scenario('Open the QAN Dashboard and verify pagination', async (I, adminPage, qa
...
+    qanPage._selectDetails(2);
+    qanPage.selectSectionInDetails(qanPage.fields.explainTabInDetails);
+    await qanPage.verifyExplainDetailsSectionDataExists();
+});
+
+Scenario('Open the QAN Dashboard and verify Explain tab in Query Details for Environment=pgsql-dev filter', async (I, adminPage, qanPage) => {
+    I.amOnPage(qanPage.url);
+    await I.waitForElement(qanPage.fields.iframe, 60);
+    adminPage.applyTimer("5m");
+    await I.switchTo(qanPage.fields.iframe); // switch to first iframe
```
Comment (puneet0191) [inline tests/verifyQAN_test.js]: when you switch to Iframe, when do you switch back?

File: tests/pages/qanPage.js [hunk 3]
```diff
@@ -125,8 +133,37 @@ module.exports = {
...
         I.click("//table/tr["+ (row + 1) + "]//td[2]");
+        this.waitForDetailsSection();
+    },
+
+    selectSectionInDetails(section) {
+        I.waitForElement(section, 30);
+        I.click(section);
+    },
+
+    waitForTableTabContentsLoaded() {
```
Comment (puneet0191) [inline tests/pages/qanPage.js]: The same feedback, I understand you want to increase readability, but having `waitForTableTabContentsLoaded` and `waitForExplainTabContentsLoaded` and then may be more in future, could you think of making a more generic function, wait for details and pass parameters? `waitForTabContentsLoaded` if it is not feasible, not a problem

File: tests/pages/pmmSettingsPage.js (hunk 1 above)
Comment (yurkovychv) [reply tests/pages/pmmSettingsPage.js]: thats for clearing field. clearField method does not work correctly with React

File: tests/verifyQAN_test.js (hunk 2 above)
Comment (yurkovychv) [reply tests/verifyQAN_test.js]: i do not need it

File: tests/pages/qanPage.js (hunk 3 above)
Comment (yurkovychv) [reply tests/pages/qanPage.js]: yes, i'll fix it as you say

**Pull request merged**